src/graphql-server/schema.graphql

```diff
@@ -139,7 +157,8 @@ type Mutation {
   holofuelAcceptOffer(transactionId: ID): Transaction
   holofuelDecline(transactionId: ID): Transaction
   holofuelCancel(transactionId: ID): Transaction
-  holofuelRecoverFunds(transaction: ID): Transaction
+  holofuelRecoverFunds(transactionId: ID): Transaction
+  holofuelRefundAllDeclined(listOfDeclinedTransactions: [CancelDeclinedTransactionsInput]): [Transaction]
```

Comment:
I think we should just be passing a list of `IDs` here. We're not using any of the other data in `CancelDeclinedTransactionsInput`.

Also, `listOf` variable naming. How about:

`holofuelRefundAllDeclined(transactionIds: [ID]): [Transaction]`

Reply:
Naming works. I'll update that now. :+1:

As mentioned above, the list of args does not though, as not all of them are primitive types.  I actually researched this solution read this one straight from the Apollo docs.  Once again, if you have other awareness, please don't hesitate to share. :pray:

Reply:
Let's jump on zoom to resolve this. Seems we're missing each other.